src/server/api/common/generate-replies-query.ts

```diff
@@ -1,22 +1,24 @@
 import { User } from '@/models/entities/user';
 import { Brackets, SelectQueryBuilder } from 'typeorm';
 
-export function generateRepliesQuery(q: SelectQueryBuilder<any>, me?: { id: User['id'] } | null) {
-	if (me == null) {
+export function generateRepliesQuery(q: SelectQueryBuilder<any>, me: any) {
+	const userId = me['id'] ?? null;
+	const showReplies: boolean = me['showTimelineReplies'] ?? false;
+	if (userId == null) {
```

Comment:
in TypeScript, `else` cause of `if (me == null)`, you can use `me` as non-null types. it means dont need this variables.
```suggestion
	if (me == null) {
```